Add `Modding/Nomination` (#8254)
ppy/osu-wiki: wiki/Modding/Nomination/en.md

---

> tags:

Reviewer: could add bubble to tags? noms are still referred as bubbles sometimes

Author: alright

  - nominations
  - nom
  - noms

>   - heart

Reviewer 2: ```suggestion
```
Hearts are typically associated with [qualification](https://osu.ppy.sh/wiki/en/Beatmap_ranking_procedure#qualification), not so much nominations.

---

# Nomination

*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*

::: Infobox
![](nomination-progress-beatmapset1857663 "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
:::

> **Nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).

Reviewer 3: ```suggestion
A **nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
```

> A **nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).

Reviewer: ```suggestion
A **nomination** is a formal vote of approval coming from [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [Ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [Ranking Criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
```

Author: hesitant to capitalize ranked as it's a regular adjective here, not the category name -- you don't say that an article is Written, or that a game is Played

Author: applied the rest

> Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are moved to the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).

Reviewer 2: Changes the order of sentences and rewords and splits some, making things more digestible. Crucially includes 2 noms/mode -> Qualified -> Ranked in the first paragraph briefly, because this is pretty core to what nominations are and do.
```suggestion
A **nomination** is a vote of approval from [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BNs) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT). It is given to completed beatmaps of sufficient quality, as judged by the nominating user. As a minimum requiement, these beatmaps must follow the [Ranking Criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype). If a beatmap reaches 2 nominations per included game mode, it enters the [Qualified category](/wiki/Beatmap/Category#qualified), and finally [Ranked](/wiki/Beatmap/Category#ranked) if no issues are found.
```

> Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are moved to the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).

Reviewer 3: ```suggestion
Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are "[qualified](/wiki/Beatmap/Category#qualified)" and enter the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).
```

Nominations may be removed from a map in one of the following cases:

- A beatmap is updated by its [host](/wiki/Beatmap/Beatmap_host).

> - Serious or unrankable issues found by a BN or a member of the NAT.

Reviewer 3: ```suggestion
- Serious or unrankable issues are found by a BN or a member of the NAT.
```

> - Serious or unrankable issues are found by a BN or a member of the NAT.

Reviewer: could add this if you want people to know the 3 kinds of nomination resets there are
```suggestion
- Serious or unrankable issues are found by a BN or a member of the NAT, which would cause them to either [reset its nomination](/wiki/People/The_Team/Beatmap_Nominators/General_Information#disqualify-or-reset-nomination), [disqualify](/wiki/People/The_Team/Beatmap_Nominators/General_Information#disqualify-or-reset-nomination) it, or place a [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto).
```

Author: this is valuable, thanks

## History

Nominations in their present form were implemented as part of the [beatmap discussion](/wiki/Beatmap_discussion) system.

> Previously, in the era of [forum modding](/wiki/Modding/Forum_modding), nominations were expressed through a combination of ![Bubble icon](/wiki/shared/icon/bubble.gif) [bubble](/wiki/Modding/Bubble) and ![Heart icon](/wiki/shared/icon/heart.gif) rank posts, where bubble icon meant the first nomination, and the heart icon was the second. Maps going for [approval](/wiki/Beatmap/Category#approved) instead of rank needed to abide by a slightly different set of rules and required three nominations, with the last of them having ![Flame icon](/wiki/shared/icon/flame.gif) flame icon.

Reviewer 3: ```suggestion
Previously, in the era of [forum modding](/wiki/Modding/Forum_modding), nominations were expressed through a combination of ![Bubble icon](/wiki/shared/icon/bubble.gif) [bubble](/wiki/Modding/Bubble) and ![Heart icon](/wiki/shared/icon/heart.gif) rank posts, where a bubble icon meant the first nomination, and the heart icon was the second. Maps going for [approval](/wiki/Beatmap/Category#approved) instead of ranking needed to abide by a slightly different set of rules and required three nominations, with the last of them having a ![Flame icon](/wiki/shared/icon/flame.gif) flame icon.
```

---
tags:
  - nominations
  - nom
  - noms
  - bubble
  - heart
---

# Nomination

*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*

::: Infobox
![](img/nomination-progress-beatmapset1857663.png "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
:::

A **nomination** is a formal vote of approval coming from [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [Ranking Criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).

> Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are "[qualified](/wiki/Beatmap/Category#qualified)" and enter the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).

Reviewer 3: ```suggestion
Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are "[qualified](/wiki/Beatmap_ranking_procedure#qualification)" and enter the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).
```
